### PR TITLE
Ruleset Engine: Update concepts, including phases

### PR DESCRIPTION
**Pull request by pedrosousa**
Addresses https://jira.cfops.it/browse/PCX-3264.

### CODE REVIEW

File: content/ruleset-engine/about/_index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
```
Comment (pedrosousa) [inline content/ruleset-engine/about/_index.md:1]: This content already existed. Moved some sections to their own pages.

File: content/ruleset-engine/about/phases.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
```
Comment (pedrosousa) [inline content/ruleset-engine/about/phases.md:1]: Previously in the About page.

File: content/ruleset-engine/about/rules.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
```
Comment (pedrosousa) [inline content/ruleset-engine/about/rules.md:1]: Previously in the About page.

File: content/ruleset-engine/about/rulesets.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
```
Comment (pedrosousa) [inline content/ruleset-engine/about/rulesets.md:1]: Previously in the About page.

File: content/ruleset-engine/reference/phases-list.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
+---
```
Comment (pedrosousa) [inline content/ruleset-engine/reference/phases-list.md:1]: New page.

File: content/ruleset-engine/about/rules.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+title: Rules
+pcx-content-type: concept
+weight: 4
+---
+
+# Rules
+
+A **rule** defines a filter and an action to perform on the incoming requests that match the filter. The rule filter **expression** defines the scope of the rule and the rule **action** defines what happens when there’s a match for the expression. Rule filter expressions are defined using the [Rules language](/ruleset-engine/rules-language/).
```
Comment (marciocloudflare) [inline content/ruleset-engine/about/rules.md]:
```suggestion
A **rule** defines a filter and an action to perform on the incoming requests that match the filter. The rule filter **expression** defines the scope of the rule and the rule **action** defines what happens when there is a match for the expression. Rule filter expressions are defined using the [Rules language](/ruleset-engine/rules-language/).
```

File: content/ruleset-engine/reference/phases-list.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+title: Phases list
+pcx-content-type: navigation
```
Comment (marciocloudflare) [inline content/ruleset-engine/reference/phases-list.md]:
```suggestion
pcx-content-type: reference
```

Comment (marciocloudflare) [reply content/ruleset-engine/reference/phases-list.md]: Seems more like a reference page. Wdyt?

**Review by marciocloudflare — APPROVED** [review]

File: content/ruleset-engine/reference/phases-list.md (hunk 7 above)
Comment (pedrosousa) [reply content/ruleset-engine/reference/phases-list.md]: Sure, my bad (copy&paste issues).

File: content/ruleset-engine/about/rules.md (hunk 6 above)
Comment (pedrosousa) [reply content/ruleset-engine/about/rules.md]: Nice catch!

File: content/ruleset-engine/reference/phases-list.md (hunk 7 above)
Comment (marciocloudflare) [reply content/ruleset-engine/reference/phases-list.md]: Thought that might be the case :)

**Pull request merged**